wiki/debugging.md

```diff
@@ -1,16 +1,18 @@
 # Debugging data issues
 
+DON'T PANIC. 
+
 Many issues can arise in the Metro galaxy, from the shallowest part of the frontend to the deepest end of the backend. Tackling data inaccuracies requires a strategic approach. This guide traces an easeful, step-by-step process for efficient debugging. 
 
-DON'T PANIC. 
+If the bug-in-question feels like a familiar one, then read below about [recurring problems](#recurring-problems).
 
 ## Identifying the point of failure
 
 [point to the charts in the overview that describe the data pipeline]
 
 Data travels along a complicated path to arrive in Metro Councilmatic. A disruption in the data pipeline can happen at four different points. 
 
-1. Legistar - the Legistar API or web interface did not provide accurate data 
+1. Legistar - the Legistar API or web interface displays inaccuracies, likely because Metro staff did not enter correct data 
```

Comment:
good clarification! this suggests that the solution likely involves human error, which i'd approach differently than if it were a website malfunction.